.github/workflows/wheel_linux_x86_64.yml

```diff
@@ -1,10 +1,12 @@
-name: Wheels
+name: Wheel::Linux::x86_64
 on:
+  release:
```

Comment:
Do we need the "types" line, as we do e.g. for `wheel_linux_ppc64le.yml`? (same question applies to some of the other workflows)

Reply:
Good catch. I think these need to be added to all.

Reply:
Good catch -- I think these need to be added to all workflows.

Reply:
Rather than changing the RC, maybe we merge this as is and modify in a subsequent PR. This won't change the wheel, and help with consistency across the wheel history. What do you think?

Reply:
Sounds good! Thanks @mlxd!